openfisca_core/tracers.py

```diff
@@ -1,9 +1,12 @@
 # -*- coding: utf-8 -*-
 
+import os
+import json
+import time
 import numpy as np
-
-from typing import List, Dict
+from typing import List, Dict, Optional
 from collections import ChainMap
+import importlib.resources as pkg_resources
```

Comment:
:exclamation: This removes the 3.6 backward compatibility

New in version 3.7.
From https://docs.python.org/3/library/importlib.html#module-importlib.resources

Reply:
Thanks @Morendil  for suggesting `pip install importlib_resources`

Reply:
However, `importlib_resources` still requires this line to be changed into
```python
import importlib_resources as pkg_resources
```